src/components/Comments.jsx

```diff
@@ -1,35 +1,35 @@
-// import { useEffect, useState } from "react"
-// import ApiRequest from "./ApiRequest"
-// import CommentsList from "./CommentsList"
+import { useEffect, useState } from "react"
+import ApiRequest from "./ApiRequest"
+import CommentsList from "./CommentsList"
 
 
-// export default function Comments({ article_id, username }) {
-//     const [comments, setComments] = useState([])
-//     const [isLoading, setIsLoading] = useState(false)
-//     const [errMsg, setErrMsg] = useState(null)
+export default function Comments({ article_id }) {
+    const [comments, setComments] = useState([])
+    const [isLoading, setIsLoading] = useState(false)
+    const [errMsg, setErrMsg] = useState(null)
 
-//     useEffect(() => {
-//         setIsLoading(true)
-//         ApiRequest(`/articles/${article_id}/comments`)
-//         .then((data) => {
-//             const { comments } = data
-//             setComments(comments)
-//             setIsLoading(false)
-//         })
-//         .catch((err) => {
-//             setIsLoading(false)
-//             const { msg } = err.response.data
-//             setErrMsg(msg)
-//         })
-//     }, [article_id])
+    useEffect(() => {
+        setIsLoading(true)
+        ApiRequest(`/articles/${article_id}/comments`)

```

Comment:
we can have a separate api call that just covers comments :) we dont need to make the api request reusable!